README.md

```diff
@@ -27,14 +27,24 @@ learning frameworks.
 ## Key Features
 
 - Support of various compression algorithms, applied during a model fine-tuning process to achieve a better performance-accuracy trade-off:
-  
+
+### Compression-Aware Training Algorithms  
+
 |Compression algorithm|PyTorch|TensorFlow|
 | :--- | :---: | :---: |
 |[Quantization](./docs/compression_algorithms/Quantization.md) | Supported | Supported |
-|[Mixed-Precision Quantization](./docs/compression_algorithms/Quantization.md#mixed_precision_quantization) | Supported | Not supported |
-|[Binarization](./docs/compression_algorithms/Binarization.md) | Supported | Not supported |
-|[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported |
-|[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported |
+|[Mixed-Precision Quantization](./docs/compression_algorithms/Quantization.md#mixed_precision_quantization) | Supported | Not supported | Not supported |
+|[Binarization](./docs/compression_algorithms/Binarization.md) | Supported | Not supported | Not supported |
+|[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported | Not supported |
+|[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported | Not supported |
+
+### Post-Training Compression Algorithms
+
+| Compression algorithm                                                       |PyTorch|TensorFlow|   ONNX   |       OpenVINO       |
+|:----------------------------------------------------------------------------| :---: | :---: |:--------:|:--------------------:|
+| [Quantization](./docs/compression_algorithms/post_training/Quantization.md) | Supported | Supported |Supported| Supported (Preview)* |
+
```

Comment:
I don't like "Supported (Preview)*". Let's state it as a "Preview" and remove the note

Reply:
I think it should be also Preview for PyTorch and TensorFlow

Reply:
Do you mean that it should be changed Supported -> Preview*  for TF, Torch and OpenVINO?

Reply:
Updated - please take a look